sdk/communication/AzureCommunicationChat/README.md

```diff
@@ -483,13 +452,12 @@ client.add(chatThreadMembers: threadMembers, chatThreadId: threadId) { result, _
 
 #### Remove a thread member
 
-Use the `removeChatThreadMember` method to remove a member from a thread.
+Use the `remove` method of `ChatThreadClient` to remove a member from a thread.
 
-- `chatThreadId` is the unique ID of the thread.
-- `chatMemberId` is the user ID in the chat thread's member list.
+- `chatParticipantId` is the user ID in the chat thread's member list.
 
 ```swift
-client.removeChatThreadMember(chatThreadId: threadId, chatMemberId: memberId) { result, _ in
+chatThreadClient.removeChatThreadMember(chatParticipantId: memberId) { result, _ in
```

Comment:
Either `remove(participant:)` or `removeParticipant(withId:)` - Note: not "removeChatThreadMemeber"